core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -268,4 +261,65 @@ public void delete(PageId pageId) throws IOException, PageNotFoundException {
   public void close() throws Exception {
     mPageStore.close();
   }
+
+  /**
+   * Attempts to add a page to the page store. The page lock must be acquired before calling this
+   * method. The metastore must be updated before calling this method.
+   *
+   * @param pageId page id
+   * @param page page data
+   * @return true if successful, false otherwise
+   */
+  private boolean addPage(PageId pageId, byte[] page) {
+    try {
+      mPageStore.put(pageId, page);
+    } catch (IOException e) {
+      LOG.error("Failed to add page {}: {}", pageId, e);
+      return false;
+    }
+    mEvictor.updateOnPut(pageId);
+    Metrics.BYTES_WRITTEN_CACHE.inc(page.length);
+    return true;
+  }
+
+  /**
+   * Attempts to delete a page from the page store. The page lock must be acquired before calling
+   * this method. The metastore must be updated before calling this method.
+   *
+   * @param pageId page id
+   * @param pageInfo page info
+   * @return true if successful, false otherwise
+   */
+  private boolean deletePage(PageId pageId, PageInfo pageInfo) {
+    try {
+      mPageStore.delete(pageId, pageInfo.getPageSize());
+    } catch (IOException | PageNotFoundException e) {
+      LOG.error("Failed to delete page {}: {}", pageId, e);
+      return false;
+    }
+    mEvictor.updateOnDelete(pageId);
+    Metrics.BYTES_EVICTED_CACHE.inc(pageInfo.getPageSize());
+    return true;
+  }
+
+  private ReadableByteChannel getPage(PageId pageId, int offset) {
```

Comment:
Add `@Nullable`

Reply:
It is annotated in the interface, is that sufficient?

Reply:
Updated, thanks.